tests/conftest.py

```diff
@@ -0,0 +1,37 @@
+import os
+from pathlib import Path
+
+import pytest
+from dotenv import load_dotenv
+from pyrdfstore import create_rdf_store
+from rdflib import Graph
+
+from pytravharv.common import QUERY_BUILDER
+from pytravharv.store import TargetStoreAccess
+
+load_dotenv()
+
+TEST_INPUT_FOLDER = Path(__file__).parent / "./inputs"
+
+
+@pytest.fixture()
```

Comment:
use the mulitple rdf_stores pattern we use already

- https://github.com/vliz-be-opsci/py-sync-fs-triples/blob/main/tests/conftest.py#L46
- https://github.com/vliz-be-opsci/py-RDF-store/blob/main/tests/conftest.py#L52